src/styles/stars.scss

```diff
@@ -0,0 +1,15 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
```

Comment:
Verify that the image path `../images/star.svg` is correct and that the image exists in the specified location. This is important to ensure the star icons display correctly.